controllers/v1/resource.js

```diff
@@ -0,0 +1,116 @@
+const resourceHelper = require("../../helper/resource")
+const {ResourceType, ResourceTypeProjection} = require("../../common/enum.utils")
+
+ /**
+     * @api {post} /mlreports/api/v1/resource/filtervalues?resourceType=program&resourceId=6013eab15faeea0e88a26ef5
+     * List of data based on collection
+     * @apiVersion 1.0.0
+     * @apiGroup public
+     * @apiSampleRequest /mlreports/api/v1/resource/filtervalues
+     * @param {json} Request-Body:
+     {
+     "projection": "block",
+        "query":{           // optional required for block
+        "districtLocationId": "b5c35cfc-6c1e-4266-94ef-a425c43c7f4e"
+        }
+      }
+     * @apiParamExample {json} Response:
+     * {
+          "message": "Program resource fetched successfully ",
+          "status": 200,
+          "result": {
+          "districts": [
+                {
+                    "id": "98ae45d7-9257-4c14-a16a-9760a442ff28",
+                    "name": "PRAKASAM"
+                },
+                {
+                    "id": "0c0391ba-610b-4796-8645-338d047b1e28",
+                    "name": "TIRUVALLUR"
+                }
+            ],
+            "organisations": [
+                {
+                    "id": "0126796199493140480",
+                    "name": "Staging Custodian Organization"
+                }
+            ]  
+          }
+     * }   
+     * @apiUse successBody
+     * @apiUse errorBody
+     */
+
+    /**
+      * List of data based on collection
+      * @method
+      * @name filtervalues
+      * @returns {JSON} list of data.
+    */
+exports.filtervalues = async function (req, res) { 
```

Comment:
@ankitpws - Remove the current ml core service API, and remove the API from KONG i.e. modify the API endpoint and downstream URL

Reply:
Also remove related functionality from code.

Reply:
@ankitpws Check if the current ML Core API does role and program level access check before returning these results. If yes, we should do it here as well.